ranges-formatter.js

```diff
@@ -0,0 +1,57 @@
+function sort(array) {
+  return array.sort(function (a, b) {
+    return a - b;
+  });
+}
+function findRanges(sortedInput) {
+  let minIndex = 0;
+  let maxIndex;
+  let rangeValueCount = 1;
+  const ranges = [];
+  sortedInput.forEach((element, index) => {
+    const difference = sortedInput[index + 1] - sortedInput[index];

```

Comment:
Will sortedInput[index+1] work fine for the last index? will not cause out of range exception or index out of bounds?